From 6c54f2daee74ed98ae42c7ee218d5b260a868c59 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Tue, 26 Mar 2024 12:25:43 +0100
Subject: [PATCH] base-passwd: Update gid for input and audio so they match
 with Android's

input and audio gid's need to be aligned with Android ones. Since webOS doesn't care about the gid, we simply patch it globally. LG uses this gid too at their end anyway, so not necessary to do this in meta-smartphone for Android only.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [webOS/LuneOS specific]

 group.master | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/group.master b/group.master
index 4c835bc..80bbdba 100644
--- a/group.master
+++ b/group.master
@@ -12,7 +12,7 @@ uucp:*:10:
 man:*:12:
 proxy:*:13:
 kmem:*:15:
-input:*:19:
+input:*:1004:
 dialout:*:20:
 fax:*:21:
 voice:*:22:
@@ -20,7 +20,7 @@ cdrom:*:24:
 floppy:*:25:
 tape:*:26:
 sudo:*:27:
-audio:*:29:
+audio:*:1005:
 dip:*:30:
 www-data:*:33:
 backup:*:34:
